### DISCUSSION TITLE
Diaporama

### DISCUSSION THREAD

**Discussion by lucile98-ode** [lucile98-ode/GPI-L2BHA-FENOUIL#37] [Q&A] [answered]
Est-ce que l'idée de mettre la vidéo de démo dans les diapo vous plait ?

Pour les diapos je pensais d'en faire pleins pour les alléger

**Comment by amrx12** [comment, marked as answer]
On devra faire une demo juste apres donc ce n'est pas forcement utile non ?

**Reply by lucile98-ode** [reply]
si on a la demo sur la diapo ca permet de le montrer durant la présentation du logiciel a la place des screens

**Reply by amrx12** [reply]
ca ne risque pas d'etre redondant ?

**Reply by HaceneM** [reply]
> On devra faire une demo juste apres donc ce n'est pas forcement utile non ?

Je suis d'accord sur le fait qu'une démo live est plus "vivant" que les démos vidéos pour les personnes assistants a la présentation.
